lib/dash/mpd_utils.js

```diff
@@ -477,20 +485,19 @@ shaka.dash.MpdUtils = class {
           // the element can be changed further.
 
           // Remove the current contents of the node.
-          while (element.childNodes.length) {
-            element.removeChild(element.childNodes[0]);
+          while (element.children.length) {
+            element.children.shift();
```

Comment:
Setting element.children to an empty array is probably more efficient than shifting all the elements.